lib/appsignal/finch.ex

```diff
@@ -44,7 +44,6 @@ defmodule Appsignal.Finch do
   defp do_finch_request_start(parent, _name, request) do
     uri = %URI{
       scheme: Atom.to_string(request.scheme),
-      path: request.path,
```

Comment:
This commit ("Remove path from Finch integration ") should leave out the path **and the query**, right?

Reply:
Ugh. You’re right, updated in https://github.com/appsignal/appsignal-elixir/pull/794/commits/7566543d7c42fb601b609d51d51262ff8cd1efef